azurerm/internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go

```diff
@@ -58,6 +60,15 @@ func resourceArmOrchestratedVirtualMachineScaleSet() *schema.Resource {
 				ValidateFunc: validation.IntBetween(0, 5),
 			},
 
+			"proximity_placement_group_id": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validate.ProximityPlacementGroupID,
+				// the Compute/VM API is broken and returns the Resource Group name in UPPERCASE :shrug:
+				DiffSuppressFunc: suppress.CaseDifference,
```

Comment:
is there an API bug tracking this so that the Compute API complies with RFC 2616?

Reply:
Well, I noticed this issue and checked the VMSS resources (they have a `proximity_placement_group_id` atttribute too) and found this comment. As a result I just copied the comment here.
Specs repo does not have an issue tracking this now. I will open a new one and update the comment here

Reply:
Issue created: https://github.com/Azure/azure-rest-api-specs/issues/10016